lib/jsonapi/plugs/transform_parameters.ex

```diff
@@ -0,0 +1,41 @@
+defmodule JSONAPI.TransformParameters do
```

Comment:
I think a better name is a good idea, `Transform` is rather generic

Reply:
One reason I kept the generic name is so that if we eventually want to add functions to handle camelCase we're in the clear. Come to think of it, though, we may want a distinct plug to deal with such a scenario.

Reply:
Let's change the name to be more specific for now. If we decide in the future to support additional functionality, we can cross that bridge then.